frontend/utils/branding.tsx

```diff
@@ -1,8 +1,10 @@
+import getConfig from 'next/config'
 import { ReactNode } from 'react'
 
 import { ClubEventType } from '../types'
 
-const site = process.env.NEXT_PUBLIC_SITE_NAME || 'clubs'
+const { publicRuntimeConfig } = getConfig()
+const site = publicRuntimeConfig.NEXT_PUBLIC_SITE_NAME || 'clubs'
```

Comment:
I feel like this should be unconditionally set to NEXT_PUBLIC_SITE_NAME, and we have the || clubs inside of the runtime config itself.